src/ScriptBuilder/PowerShellCounterWriter.cs

```diff
@@ -6,7 +6,7 @@
 
     static class PowerShellCounterWriter
     {
-        public static void WriteScript(string scriptPath, IEnumerable<TimerDefinition> timers, IEnumerable<MeterDefinition> meters, Dictionary<string, string> legacyInstanceNameMap)
+        public static void WriteScript(string scriptPath, IEnumerable<DurationDefinition> timers, IEnumerable<SignalDefinition> meters, Dictionary<string, string> legacyInstanceNameMap)
```

Comment:
The change should affect not only types but also variable/parameter names. It's hard to read timers and think durations. This is not the same.